examples/java/example_maven/pom.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I *think* this `pom.xml` is the route we want to go for publish our "kotlin" package? When you have time let's sync on getting maven setup/pom.xml for the kotlin sdk and rtl packages so I can connect up release-please